src/BudgetManager/BudgetManager.CQRS/Handlers/NotificationHandlers/GetNotificationListHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using BudgetManager.CQRS.Queries.NotificationQueries;
+using BudgetManager.Model;
+using BudgetManager.Shared.DataAccess.MongoDB.BaseImplementation;
+using MediatR;
+
+namespace BudgetManager.CQRS.Handlers.NotificationHandlers
+{
+    public class GetNotificationListHandler : IRequestHandler<GetNotificationListQuery, IEnumerable<Notification>>
```

Comment:
Shouldn't 'NotificationResponse' be used instead of 'Notification'?

Reply:
I don't feel like it is necessary but sure, I can change it.